dataactcore/migrations/versions/29600f7fe065_add_agency_fields_to_published.py

```diff
@@ -0,0 +1,49 @@
+"""Add agency fields to published
+
+Revision ID: 29600f7fe065
+Revises: f15485f0092b
+Create Date: 2017-05-14 20:21:37.159452
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '29600f7fe065'
+down_revision = 'f15485f0092b'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_data_broker():
+    op.add_column('detached_award_financial_assistance', sa.Column('awarding_agency_name', sa.Text(), nullable=True))
+    op.add_column('detached_award_financial_assistance', sa.Column('awarding_sub_tier_agency_n', sa.Text(), nullable=True))
+    op.add_column('detached_award_financial_assistance', sa.Column('funding_agency_name', sa.Text(), nullable=True))
+    op.add_column('detached_award_financial_assistance', sa.Column('funding_sub_tier_agency_na', sa.Text(), nullable=True))
```

Comment:
These shouldn't be adding to the detached one, we aren't asking them to include the title in their file, we're deriving it on submit